frontend/.env.development

```diff
@@ -14,6 +14,9 @@ SENDY_API_KEY=
 SENDY_API_URL=
 SENDY_LIST_ID=
 
+# when running the full containerized app locally in Docker, uncomment the line below to
+# enable connection to the conatainerized API server
+# API_URL=http://host.docker.internal:8080
```

Comment:
We have a similar issue in the API where the path we want inside docker and outside are different.

I use direnv to handle env vars with an `.envrc` file that auto-loads when I'm in the API directory. The local.env file always has the docker specific values, and I override for non-docker values outside.

```sh
#!/bin/bash

# This file can be used by direnv to load the local.env file
# into your current terminal session.
# Steps:
# * Install direnv: https://direnv.net/
# * Configure direnv: https://direnv.net/docs/hook.html
# * In this folder, run "direnv allow ."

set -o allexport
source local.env
set +o allexport

# If you are running outside of the Docker container, the DB can
# be found on localhost:5432. Inside the container it's referenced via
# the name of the docker container.
export DB_HOST=localhost
export S3_ENDPOINT_URL=http://localhost:4566
export PY_RUN_APPROACH=local

export SEARCH_ENDPOINT=localhost
```

I thought this was somewhere in the docs, but it looks like it wasn't copied over from the template repo: https://github.com/navapbc/template-application-flask/tree/main/docs/app#running-natively

Reply:
Cool, and very good to know. This could be an option for the frontend as well, but the common use case I think is the opposite of the API side. Where on the API side it seems like we're usually running the DB in a container so that is the default value, on the frontend side we're usually running the app outside of the container so that would be the default value to commit.

Also, with an .envrc file pointing to localhost, for example, how do you deal with the case where you want to run the DB in a container? Do you unload direnv before starting up the containers? I'm having a hard time figuring out an easy system for this that doesn't require some action on the part of the developer when they're going with the less common use case, whether it's uncommenting code or disabling a plugin...

Reply:
Most things always run inside a container, there isn't a way to have the DB run outside the container in our setup.

What running locally/non-locally mostly changes is for how scripts/tests work. If I'm running the python tests in the docker container, when they hit the DB, it needs to work one way, but outside the docker container, I want a different URL.

.envrc never affects anything with Docker, just sometimes tells our scripts to run differently outside Docker. This mostly matters for the `PY_RUN_APPROACH` flag which tells some of our make commands to run natively rather than inside docker.